Project Files/Calculator/CyAcid.java

```diff
@@ -26,94 +26,94 @@ else if(currCyAcid < 30) {
             String cyAcidString = Double.toString(currCyAcid);
             switch (cyAcidString) {
                 case "0":
-                    totalAmount = volumeFactor * 30;
+                    totalAmount = (volumeFactor * amtNeeded) * 30;
```

Comment:
This was the logic error that was fixed with this update. Before it was just multiplying the volume factor by the ppm increase amount. Now it actually does the correct calculation. I'm thinking of changing this to a loop like I did in the Alkalinity class. The up side of doing the switch statement is that it will be much faster than a loop, but it has less range with higher instruction count. The up side of a loop is that you can have huge range with low instruction count, but it will be slower.